main.go

```diff
@@ -175,3 +184,54 @@ func getEnv(key string, defaultVal string) string {
 	}
 	return defaultVal
 }
+
+func isOnCloudFoundry() bool {
+	return getEnv("VCAP_APPLICATION", "") != ""
+}
+
+func getCloudFoundryRedisBindings() ([]string, []string, []string) {
+	if !isOnCloudFoundry() {
+		return []string{}, []string{}, []string{}
+	}
+
+	var addrs []string
+	var passwords []string
+	var aliases []string
+
+	appEnv, _ := cfenv.Current()
```

Comment:
Does this need an `err` check? Would line 203 panic with nil reference if it fails?

Reply:
We already checked if we are on a CF environment, so err might not be interesting here. I will take another look to be sure.